### ISSUE TITLE
Community, topics and different languages problem

### ISSUE THREAD

**Issue by dmitryzuev** [scott/helpy#36]
Hi there.
Just found about your app, and it looks great!

Now about the problem:
When you looking at the demo community http://demo.helpy.io/ru/community
under other than english language (Russian, for example), you get error if you want to look at some topic.

Error: http://demo.helpy.io/ru/topics/24-hacking-hard-drive-is-not-working-/posts
Success: http://demo.helpy.io/en/topics/24-hacking-hard-drive-is-not-working-/posts

Bug replicates also here:
http://demo.helpy.io/ru/community/3-Questions-and-Answers/topics

But Feature Requests works fine
http://demo.helpy.io/ru/topics/28-synthesizing-program-initial-questions/posts

I couldn't understand where is the problem, but probably it's because I hadn't install helpy yet.

**Comment by scott** [comment]
Hi Dmitry,

Thanks for your detailed issue report!  Ironically I discovered this today and was working on resolving it.  The cause of the problem is in the Russian translation file with pluralization- as you probably know, english does not use as many conjugations for pluralization as in Russian- so the translation file typically uses one and other as keys, but with Russian it takes zero, one, few and many.  I didn't realize this when I originally set up the translation file- so it uses the wrong keys and is generating errors as a result.  I started fixing this today and will push a new version tomorrow.  It also has some more of the translation complete.

Scott
